backend/modules/auth/provider.py

```
@@ -0,0 +1,454 @@
+        db.commit()
+        db.refresh(user)
+        return user
+    
+    def create_user_login_response(self, user: User):
+        """Create login response for OAuth user"""
+        from modules.auth.schemas import UserLoginResponse
+        from common.db.schemas import UserResponse
+        
+        return UserLoginResponse(
+            access_token="",
+            token_type="cookie",
```

Ruff (0.14.6)

Possible hardcoded password assigned to argument: "token_type"

(S106)